reddit_service_websockets/dispatcher.py

```diff
@@ -22,6 +46,14 @@ def __init__(self, metrics):
     def on_message_received(self, namespace, message):
         consumers = self.consumers.get(namespace, [])
 
+        # Compress the message
+        compressed = make_compressed_frame(message, COMPRESSOR)
+        message = Message(
+            compressed=compressed,
+            raw=message,
+            percent_compressed=1 - float(len(compressed))/float(len(message)),
```

Comment:
Since this is computable downstream given the other two fields in `Message`, it feels a bit odd to compute it up front here far away from the actual stats code that it's for. In other words: why should this code care that the stats code is gonna want this value?